rich/pretty.py

```diff
@@ -272,7 +272,7 @@ def __call__(self, value: Any) -> Any:
         # replace plain text formatter with rich formatter
         rich_formatter = RichFormatter()
         ip.display_formatter.formatters["text/plain"] = rich_formatter
-    except Exception:
```

Comment:
While I never used nuitka, I know enough to solute removal of "except Exception" ;)